health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BoundarySummary.js

```
@@ -0,0 +1,188 @@
+  const history = useHistory();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  const searchParams = new URLSearchParams(location.search);
+  const id = searchParams.get("id");
+  const noAction = searchParams.get("action");
+  const [showToast, setShowToast] = useState(null);
+  const isPreview = searchParams.get("preview");
+  const [currentStep, setCurrentStep] = useState(2);
+  const currentKey = searchParams.get("key");
+  const [key, setKey] = useState(() => {
+    const keyParam = searchParams.get("key");
+    return keyParam ? parseInt(keyParam) : 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)